### PR TITLE
Doc changes for partial+startup

### PR DESCRIPTION
**Pull request by KSchoenleber**
I made some changes in the documentation of the startup and partila feature, which make it more comprehensible in my opinion. I also made the .rst more readable in the editor by introducing line ends after 80 signs in (almost) each line.

### CODE REVIEW

File: doc/mathdoc/equ-process.rst [hunk 1]
```diff
@@ -3,149 +3,229 @@
...
+capacity is defined by the constraint as the sum of the parameter process 
+capacity installed :math:`K_{vp}` and the variable new process capacity 
+:math:`\hat{\kappa}_{vp}`. In mathematical notation this is expressed as: 
 
 .. math::
 
 	\forall v\in V, p\in P\colon\ \kappa_{vp} = K_{vp} + \hat{\kappa}_{vp}
 
-In script ``urbs.py`` the constraint process capacity rule is defined and calculated by the following code fragment:
+In script ``urbs.py`` the constraint process capacity rule is defined and
```
Comment (ojdo) [inline doc/mathdoc/equ-process.rst]: Better: In file ``model.py``, the constraint...

File: doc/mathdoc/equ-process.rst [hunk 2]
```diff
@@ -3,149 +3,229 @@
...
+ratio :math:`r_{pc}^\text{in}`. In mathematical notation this is expressed as:
 
 .. math::
 
-	\forall v\in V, p\in P, t\in T_m\colon\ \epsilon^\text{in}_{vcpt} = \tau_{vpt} r^\text{in}_{pc}
+	\forall v\in V, p\in P, t\in T_m\colon\ \epsilon^\text{in}_{vcpt} =
+    \tau_{vpt} r^\text{in}_{pc}
 
-In script ``urbs.py`` the constraint process input rule is defined and calculated by the following code fragment:
+In script ``urbs.py`` the constraint process input rule is defined and
```
Comment (ojdo) [inline doc/mathdoc/equ-process.rst]: See last comment.

File: doc/mathdoc/equ-process.rst [hunk 3]
```diff
@@ -3,149 +3,229 @@
...
+ratio :math:`r_{pc}^\text{out}`. In mathematical notation this is expressed as:
 
 .. math::
 
-	\forall v\in V, p\in P, t\in T_m\colon\ \epsilon^\text{out}_{vpct} = \tau_{vpt} r^\text{out}_{pc}
+	\forall v\in V, p\in P, t\in T_m\colon\ \epsilon^\text{out}_{vpct} =
+    \tau_{vpt} r^\text{out}_{pc}
 
-In script ``urbs.py`` the constraint process output rule is defined and calculated by the following code fragment:
+In script ``urbs.py`` the constraint process output rule is defined and
```
Comment (ojdo) [inline doc/mathdoc/equ-process.rst]: ...

File: doc/mathdoc/equ-process.rst [hunk 4]
```diff
@@ -3,149 +3,229 @@
...
+notation this is expressed as:
 
 .. math::
 
-	\forall v\in V, p\in P, c\in C_\text{sup}, t\in T_m\colon\ \epsilon^\text{in}_{vpct} = \kappa_{vp} s_{vct}
+	\forall v\in V, p\in P, c\in C_\text{sup}, t\in T_m\colon\
+    \epsilon^\text{in}_{vpct} = \kappa_{vp} s_{vct}
 
-In script ``urbs.py`` the constraint intermittent supply rule is defined and calculated by the following code fragment:
+In script ``urbs.py`` the constraint intermittent supply rule is defined and
```
Comment (ojdo) [inline doc/mathdoc/equ-process.rst]: ... (last comment for this one)

File: doc/mathdoc/equ-process.rst (hunk 1 above)
Comment (KSchoenleber) [inline doc/mathdoc/equ-process.rst]: I made these changes in my other pull request. I did not tidy up the seperate requests properly, sorry.

Comment (ojdo) [inline doc/mathdoc/equ-process.rst]: As long as the changes converges towards "more correct", everything's fine :+1:

**Pull request merged**